packages/autocannon-ui-backend/routes/execute.js

```diff
@@ -18,7 +18,9 @@ const schema = {
 }
 
 module.exports = async function (fastify) {
+  fastify.register(require('fastify-cors'), { origin: '*' })
   fastify.post('/api/execute', { schema }, function (request, reply) {
+    reply.raw.setHeader('Access-Control-Allow-Origin', '*')
```

Comment:
Would this not work with fastify-cors alone? If so, why?

Reply:
It doesn't, after trying some approaches for some reason we need to specify `Access-Control-Allow-Origin` on POST to make it work without CORS issue.
Also I opened a question regarding it https://github.com/fastify/help/issues/619

Reply:
Understood!